indico/modules/events/abstracts/blueprint.py

```diff
@@ -75,5 +76,6 @@
                  RHPreviewEmailTemplate)
 
 # Abstract-specific management
-_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/', 'manage_abstract', RHManageAbstract)
+_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/', 'manage_abstract', RHManageAbstract, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/reset', 'reset_abstract_jugment', RHResetAbstractJudgment)
```

Comment:
:warning: should be POST-only unless you intend to show a pretty confirmation prompt (like on event deletion or certain category operations)